src/main/java/io/mathdojo/useraccountservice/services/IdentityService.java

```diff
@@ -197,11 +201,17 @@ public String toString() {
         return sb.toString();
     }
 
+    @Repository
+    public interface MathDojoUserRepository extends MongoRepository<User, String> {
```

Comment:
Should this be in its own class? I'm not sure how this and auto wiring would work together?